components/press-logos/national-geographic.js

```diff
@@ -0,0 +1,128 @@
+import { useThemeUI } from 'theme-ui'
+
+const Logo = () => {
+  const context = useThemeUI()
+  const { primary } = context.theme.colors
+
+  return (
+    <svg viewBox='0 0 150 100' width='100%' height='100%'>
+      <style type='text/css'>
+        {`
+  .st0{fill:${primary};}
+`}
+      </style>
+      <g>
+        <path
+          className='st0'
```

Comment:
Reminder (for both of us!) that we're using the same class `st0` across all the logos... if the styles for any of the logos we're different, we'd risk unintentionally overriding styles, but because they are all the same it doesn't matter. Fine to leave for now, but we might want to fix in the future. In an ideal world, our SVG -> React pipeline would create new, unique classes for every file.